src/pocketmine/inventory/transaction/action/CreativeInventoryAction.php

```diff
@@ -33,7 +33,7 @@ class CreativeInventoryAction extends InventoryAction{
 	 */
 	const TYPE_DELETE_ITEM = 0;
 	/**
-	 * Player took an item from the creative window.
+	 * Player took an item from the creative window or signs a book.
```

Comment:
this can be removed

Reply:
It does still get sent for creative mode book signing though.

Reply:
yes, but you're not using that, and it's likely that that is a bug and not intended (mismatch in survival)